application/views/themeOptions/options_core.php

```diff
@@ -121,8 +121,13 @@
                                 $optionsValues = !empty($attribute['options']) ? explode('|', $attribute['options']) : array();
                                 $optionLabels = !empty($attribute['optionlabels']) ? explode('|', $attribute['optionlabels']) : array();
                                 $options = array_combine($optionsValues, $optionLabels);
-                                if ($bInherit && isset($sParentOption)){
-                                    $options['inherit'] = gT("Inherit") . ' [' . $sParentOption . ']';
+                                if ($bInherit && isset($sParentOption)) {
+                                    if(is_numeric($sParentOption) && array_key_exists($sParentOption, $options)) {
```

Comment:
Why is_numeric ?

Reply:
That;s how we expect the `sParentOption` to come like.
If it comes like that, we map it. If not, we show it as it is coming.

Reply:
For Show popusp : OK https://github.com/LimeSurvey/LimeSurvey/blob/096476f68a8edf1f2eea9bce4ba5817ef6b09238/themes/survey/vanilla/config.xml#L63

But for all other : https://github.com/LimeSurvey/LimeSurvey/blob/096476f68a8edf1f2eea9bce4ba5817ef6b09238/themes/survey/vanilla/config.xml#L65 

We need to translate too …

Reply:
Another reason to not use hungarian notation, lol. `$sParentOption`, s = string, but is_numeric check? ^^